app/views/macros/showExamples.njk

```diff
@@ -31,7 +31,7 @@
     <div class="app-component-preview">
       <iframe
       class="js-component-preview app-component-preview__iframe"
-      src="{{ path }}?iframe=true"
+      src="{{ path }}?{{ legacy_query }}iframe=true"
```

Comment:
This will result in e.g. `/components/accordion/preview?legacy=1iframe=true` – there's no `&` joining the two parameters together.

Reply:
thanks it is on my todo I just wanted to push up what I have done so far ;)